### PR TITLE
my syntax fixes

### PR DESCRIPTION
**Pull request by gertadam**

### CODE REVIEW

File: dynamic_nn.py [hunk 1]
```diff
@@ -1,24 +1,26 @@
...
-        self.input_layer = 2 * random.random((no_input_notes, no_notes_in_hidden_layers)) - 1
+        self.input_layer = 2 * random.random((no_input_nodes, no_nodes_in_hidden_layers)) - 1
 
         self.hidden_layers = []
-
-        for layer in xrange(no_hidden_layers):
-            self.hidden_layers.append(2 * random.random((no_notes_in_hidden_layers, no_notes_in_hidden_layers)) - 1)
+        
+        layer=0
+        while layer <= no_hidden_layers: #in xrange(no_hidden_layers):
```
Comment (benjaco) [inline dynamic_nn.py]: please use range for readability

File: dynamic_nn.py [hunk 2]
```diff
@@ -29,66 +31,75 @@ def __sigmoid_derivative(self, x):
         return x * (1 - x)
 
     def train(self, traning_inputs, traning_outputs, traning_interations):
-
-        for iteration in xrange(traning_interations):
```
Comment (benjaco) [inline dynamic_nn.py]: please use range again

File: dynamic_nn.py [hunk 3]
```diff
@@ -29,66 +31,75 @@ def __sigmoid_derivative(self, x):
...
-
+        num_lines = 10
+        devider = (traning_interations/num_lines)  #this will gives us 10 lines running through the interations 
+        #
+        iteration = 0
+        while iteration <= traning_interations:
```
Comment (benjaco) [inline dynamic_nn.py]: remove those lines and use range again

File: dynamic_nn.py [hunk 4]
```diff
@@ -29,66 +31,75 @@ def __sigmoid_derivative(self, x):
...
+            
             # adjust the weights (backpropagation)
             self.input_layer += dot(traning_inputs.T, errors[0].T)
-            for i in xrange(len(self.hidden_layers)):
+            for i in range(len(self.hidden_layers)):
                 self.hidden_layers[i] += dot(layers[i].T, errors[i + 1].T)
-
-
+            #            
+            if (iteration / devider) == 0:
```
Comment (benjaco) [inline dynamic_nn.py]: use this insted
`iteration % 10 == 0`

File: dynamic_nn.py [hunk 5]
```diff
@@ -29,66 +31,75 @@ def __sigmoid_derivative(self, x):
...
-    no_output_notes=1
-)
+n_net = NeuralNetwork(no_feat, no_output, no_hidden_layers, no_nodes_in_hidden_layers)
 
-print len(neural_network.input_layer)
-for x in neural_network.hidden_layers:
-    print len(x[0])
+#header
+print ("num features:",len(n_net.input_layer))
+x=0
```
Comment (benjaco) [inline dynamic_nn.py]: why print out 0

File: dynamic_nn.py [hunk 6]
```diff
@@ -29,66 +31,75 @@ def __sigmoid_derivative(self, x):
...
+print ('x:',x)
 
-print "Train"
+for x in n_net.hidden_layers:
+    print ('x[0]:',x[0])
```
Comment (benjaco) [inline dynamic_nn.py]: what is being printet out here? looks like it it the weights in the first neron in each layer, why ?

File: dynamic_nn.py (hunk 5 above)
Comment (gertadam) [inline dynamic_nn.py]: I couldn't get the print(len(x(0)) syntax to work, so I just wrote out what would work - the intention was to remove the syntax error first  then make the print work subsequently

File: dynamic_nn.py (hunk 6 above)
Comment (gertadam) [inline dynamic_nn.py]: I couldn't get the print(len(x(0)) syntax to work, so I just wrote out what would work - the intention was to remove the syntax error first then make the print work subsequently

Comment (benjaco) [inline dynamic_nn.py]: i think following will work
`print ('Layer width: ',str(len(x))])`

**Comment by gertadam** [comment]
I'm confused now - where do I put my file with you requested changes ?

File: dynamic_nn.py (hunk 6 above)
Comment (gertadam) [inline dynamic_nn.py]: OK I will put you suggestion into my code

**Comment by benjaco** [comment]
"Just add a new commit to the branch used in the pull request and push the branch to GitHub. The pull request will automatically be updated with the additional commit."
http://stackoverflow.com/questions/7947322/preferred-github-workflow-for-updating-a-pull-request-after-code-review

**Pull request merged**